stan/math/prim/mat/fun/accumulator.hpp

```diff
@@ -34,7 +34,7 @@ class accumulator {
   /**
    * Destroy an accumulator.
    */
-  ~accumulator() {}
```

Comment:
Is there a reason for defining the accumulator destructor as empty here? tmk this still calls the destructor for all the accumulates members